typeDefinition/modifiers.d.ts

```diff
@@ -110,6 +110,10 @@ export interface VisibleModifier<T> {
 	visible(): T;
 }
 
+export interface IsLoadedModifier<T> {
+	isLoaded(): T;
```

Comment:
Please reflect this changes in
testDefinition/examplesTS/elementChain.ts

Reply:
Done